index.js

```diff
@@ -19,36 +19,37 @@ function GarageDoorOpener(log, config) {
     this.targetDoorState = TargetDoorState.CLOSED;
     this.invertDoorState = defaultVal(config["invertDoorState"], false);
     this.invertSensorState = defaultVal(config['invertSensorState'], false);
-	this.default = defaultVal(config["default_state"], false);
-	this.duration = defaultVal(config["duration_ms"], 0);
+    this.default = defaultVal(config["default_state"], false);
+    this.duration = defaultVal(config["duration_ms"], 0);
+    this.pullConfig = defaultVsl(config["input_pull"], "none");
```

Comment:
Please correct "defaultVsl".  It should be "defaultVal".

Reply:
Oops. Fixed.